src/groupInfo/components/GroupGalleryUI.vue

```diff
@@ -8,12 +8,17 @@
       :filtered-my-groups="filteredMyGroups"
       :filtered-other-groups="filteredOtherGroups"
       :expanded="expanded"
-      :my-coordinates="myCoordinates"
+      :my-coordinates="false"
```

Comment:
Checking out this branch and running it on my machine I get the following error in the console

```
[Vue warn]: Invalid prop: type check failed for prop "myCoordinates". Expected Object, got Boolean with value false.

found in

---> <GroupGalleryMap> at src/groupInfo/components/GroupGalleryMap.vue
       <GroupGallery> at src/groupInfo/components/GroupGalleryUI.vue
         <ConnectGroupGallery> at src/groupInfo/pages/GroupGallery.vue
           <QPage>
             <QPageContainer>
               <QLayout>
                 <MainLayout> at src/base/pages/MainLayout.vue
                   <App> at src/App.vue
                     <Root>
```

Reply:
Ah, now I see I just changed it during dev to simplify the logic, reverted it back to how it is supposed to be now.